src/__forks__/Promise.native.js

```diff
@@ -0,0 +1,35 @@
+/**
+ *
+ * Copyright 2013-2014 Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Fixed in https://github.com/skevy/fbjs/commit/7b58cbe2b6d6d99fa315a514f91480df1362d463